relay-server/src/services/processor/ourlog.rs

```diff
@@ -112,13 +128,54 @@ fn scrub(
     Ok(())
 }
 
+#[cfg(any(feature = "processing", test))]
+fn process_attribute_types(ourlog: &mut OurLog) {
+    let Some(attributes) = ourlog.attributes.value_mut() else {
+        return;
+    };
+
+    let attributes = attributes.iter_mut().map(|(_, attr)| attr);
+
+    for attribute in attributes {
+        use OurLogAttributeType::*;
+
+        let Some(inner) = attribute.value_mut() else {
+            continue;
+        };
+
+        match (&mut inner.value.ty, &mut inner.value.value) {
```

Comment:
I scrapped the value conversion string -> double and string -> integer, as discussed. Let's start strict.